x-pack/plugins/cases/public/components/case_action_bar/actions.tsx

```diff
@@ -84,7 +84,7 @@ const ActionsComponent: React.FC<CaseViewActions> = ({ caseData, currentExternal
 
   return (
     <EuiFlexItem grow={false} data-test-subj="case-view-actions">
-      <PropertyActions propertyActions={propertyActions} />
+      <PropertyActions propertyActions={propertyActions} customDataTestSubj={'case'} />
```

Comment:
For the single case view we pass in the `case` value so the data test subject ellipses is unique